docs/users/Versioning.mdx

```diff
@@ -4,75 +4,7 @@ title: Versioning
 ---
 
 We follow [semantic versioning (semver)](https://semver.org).
-This page exists to help set guidelines around when what we consider to fall within each of the semver categories.
-
-All of the packages in this project are published with the same version number to make it easier to coordinate both releases and installations.
-
-We publish a canary release on every successful merge to `main`, so **you never need to wait for a new stable version to make use of any updates**.
```

Comment:
The idea of a canary release is mentioned in the _Releases_ page, but the emphasis on _never need to wait for a new stable version_ I don't see anywhere else. Do you?